disneyApi/src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,88 @@
+import { useContext } from 'react';
+import { ThemeContext } from '../../context/ThemeContext';
+import {
+	ModalImg,
+	ModalList,
+	ModalLists,
+	ModalContent,
+	ModalBackground,
+} from './';
+import { Button } from '../';
+import useEscKey from '../../hooks/useEscKey';
+
+const Modal = ({ setIsModalShown, data }) => {
+	const { themeColors } = useContext(ThemeContext);
+
+	const dataEntries = Object.entries(data);
+	const filteredDataEntries = dataEntries.filter(
+		(item) =>
+			(item[0] === 'films' ||
+				item[0] === 'shortFilms' ||
+				item[0] === 'videoGames' ||
+				item[0] === 'tvShows') &&
+			item[1].length
+	);
+
+	const closeModalByBackgroundClick = (e) => {
```

Comment:
Consider renaming closeModalByBackgroundClick to handleBackgroundClick for consistency with event handler naming conventions.